src/app/lib/config/services/arabic.js

```diff
@@ -1,110 +1,114 @@
-import { latin } from '@bbc/gel-foundations/scripts';
+import { arabic } from '@bbc/gel-foundations/scripts';
 import { news as brandSVG } from '@bbc/psammead-assets/svgs';
 
 const service = {
-  lang: `ar-SA`,
-  articleAuthor: `http://www.facebook.com/bbcarabic`,
-  articleTimestampPrefix: 'Updated',
+  lang: 'ar',
+  product: 'BBC News',
+  articleAuthor: 'https://www.facebook.com/bbcnews',
+  articleTimestampPrefix: 'جدّد في',
   atiAnalyticsAppName: 'news-arabic',
-  brandName: 'BBC News Arabic',
-  product: 'BBC News Arabic',
+  brandName: 'BBC News عربي',
+  serviceLocalizedName: 'عربي',
   defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/arabic.png',
-  defaultImageAltText: 'BBC News Arabic',
-  dir: `ltr`,
-  externalLinkText: ', external',
-  imageCaptionOffscreenText: 'Image caption, ',
-  videoCaptionOffscreenText: 'Video caption, ',
-  audioCaptionOffscreenText: 'Audio caption',
-  defaultCaptionOffscreenText: 'Caption, ',
-  imageCopyrightOffscreenText: 'Image source, ',
-  locale: `ar-SA`,
-  datetimeLocale: `ar-SA`.toLowerCase(),
+  defaultImageAltText: 'BBC News عربي',
+  dir: 'rtl',
+  externalLinkText: 'خارجي ،',
+  imageCaptionOffscreenText: '،التعليق على الصورة',
+  videoCaptionOffscreenText: '،التعليق على الفيديو',
+  audioCaptionOffscreenText: '،التعليق على التسجيل السماعي',
+  defaultCaptionOffscreenText: '،التعليق',
+  imageCopyrightOffscreenText: '،مصدر الصورة',
+  locale: 'ar',
+  datetimeLocale: 'ar',
   service: 'arabic',
-  serviceName: 'News Arabic',
+  serviceName: 'Arabic',
   themeColor: `${C_POSTBOX}`,
   twitterCreator: '@BBCArabic',
   twitterSite: '@BBCArabic',
   noBylinesPolicy: 'https://www.bbc.com/news/help-41670342#authorexpertise',
   publishingPrinciples: 'https://www.bbc.com/news/help-41670342',
-  script: latin,
+  script: arabic,
   manifestPath: '/articles/manifest.json',
-  swPath: '/articles/sw.js',
+  swPath: '/sw.js',
   translations: {
     error: {
       404: {
         statusCode: '404',
-        title: '404 الصفحة غير موجودة',
-        message: 'نرجو التأكد من اسم الموقع',
+        title: 'لم يتسن العثور على الصفحة',
+        message:
+          'آسفون، لم نتمكن من العثور على الصفحة التي تطلبها. الرجاء حاول:',
         solutions: [
-          'Double checking the url',
-          'Hitting the refresh button in your browser',
-          'Searching for this page using the BBC search bar',
+          'أن تتأكد من عنوان الصفحة (url)',
+          'أن تنقر زر التجديد (refresh) في برنامج التصفح الذي تستخدمه',
+          'أن تبحث عن هذه الصفحة باستخدام خانة البحث الخاصة ببي بي سي',
         ],
-        callToActionFirst: 'Alternatively, please visit the ',
-        callToActionLinkText: 'الصفحة الرئيسية لأخبار بي بي سي',
+        callToActionFirst: 'كبديل، يرجى زيارة ',
+        callToActionLinkText: 'الصفحة الرئيسية عربي لأخبار بي بي سي',
         callToActionLast: '',
         callToActionLinkUrl: 'https://www.bbc.com/arabic',
       },
       500: {
         statusCode: '500',
```

Comment:
```suggestion
        statusCode: '۵۰۰',
```

Reply:
Resolving for this reason: https://github.com/bbc/simorgh/pull/2706#discussion_r307236205